.github/workflows/security-tests.yml

```
@@ -0,0 +1,369 @@
+    
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Download all artifacts
+        uses: actions/download-artifact@v4
+        with:
+          path: security-artifacts
+
+      - name: Generate security report
+        run: |
```

actionlint (1.7.7)

property "console_count" is not defined in object type {}

(expression)